TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -161,26 +161,28 @@ class AllStopsTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.busIdentifier, for: indexPath) as! BusStopCell
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.placeIdentifier, for: indexPath) as! PlaceTableViewCell
```

Comment:
More of a general comment but should we go into all of the cellForRowAt functions that have this force casting for cells and change them to the `guard let cell = dequeue... else { return UITableViewCell() }`

Reply:
This actually is not necessary as this should always cast properly.